### PR TITLE
Setup CI and badges

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
with cask-package-toolset ;)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  fast_finish: true
+  allow_failures:
+    - env: EVM_EMACS=emacs-git-snapshot
+
+sudo: required
+before_install:
+  - sudo apt-get install jq
+  - curl -fsSkL https://gist.github.com/rejeep/7736123/raw > travis.sh && source ./travis.sh
+  - export PATH="/home/travis/.cask/bin:$PATH"
+  - export PATH="/home/travis/.evm/bin:$PATH"
```
Comment (rejeep) [inline .travis.yml]: These two lines are already in the script: https://gist.githubusercontent.com/rejeep/7736123/raw

**Comment by AdrieanKhisbe** [comment]
I didn't have noticied that I made cpt with the older gist in mind.

As a result:
- I updated [cask-package-toolset](https://github.com/AdrieanKhisbe/cask-package-toolset.el)
- I put coverage for jq :)

**Comment by rejeep** [comment]
Finally got around to adding this! Thanks! :)

**Comment by AdrieanKhisbe** [comment]
You're welcome @rejeep :)

**Pull request merged**